src/three-components/ModelScene.js

```diff
@@ -110,10 +98,9 @@ export default class ModelScene extends Scene {
     this.shadowLight.position.set(0, 10, 0);
     this.shadowLight.name = 'ShadowLight';
 
-    this.camera = new PerspectiveCamera(FOV, this.aspect, 0.1, 100);
+    this.camera = new PerspectiveCamera(45, this.aspect, 0.1, 100);
```

Comment:
Well, I moved the FOV default const to controls.ts. These values are really never used, but required just to allocate a camera. Probably deserves a comment.

Reply:
We tend to document constant values like this in the form `const NOT_MAGIC_FOR_THIS_REASON = 1434.7` in the global module scope. It would be fine to export something like `DEFAULT_FOV` from [our "constants" module](https://github.com/GoogleWebComponents/model-viewer/blob/master/src/constants.ts).

Reply:
It feels wrong to export a const that affects nothing. Besides, the precedent was already set with the near and far plane, which also affect nothing (these values are always overwritten before a model is rendered). I'll put a comment.

Reply:
Well, to be clear: it's not wrong to do that. And, a bad precedent is not an excuse to make the code less readable.